### PR TITLE
fix quietcoolwu name

### PR DESCRIPTION
**Pull request by quietcoolwu**
/kind documentation

### CODE REVIEW

**Comment by tpepper** [comment]
We have an additional problem here.  You commit has its "Author:" set to something different that is a CLA signer.  I can help you with this.

**Comment by tpepper** [comment]
🎉

File: shanghai/OWNERS [hunk 1]
```diff
@@ -18,7 +18,7 @@ reviewers:
   - m13m
   - indigophillips
   - ewt_tang
-  - quietcoolwn
+  - quietcoolwu
```
Comment (quietcoolwu) [inline shanghai/OWNERS]: Test comment

**Pull request merged**